site.yml.sample

```diff
@@ -1,52 +1,88 @@
 ---
 # Defines deployment design and assigns role to server groups
 
+- hosts: all 
+  gather_facts: false
+  tasks:
+    # If we can't get python2 installed before any module is used we will fail
+    # so just try what we can to get it installed
+    - name: Check for python2
+      stat:
+        path: /usr/bin/python
+      ignore_errors: yes 
+      register: systempython2
+    # Try to install python2 on Ubuntu Xenial and possibly others
+    - name: Install python2 for Ubuntu
+      raw: sudo apt-get -y install python-simplejson
+      ignore_errors: yes 
+      when: systempython2.stat.exists is undefined or systempython2.stat.exists == false
```

Comment:
No can do I'm afraid. This is running before we have run "setup" and defined these variables. Unfortunately that's the point, if we run "setup" withput python2 installed we fail so my intention was just attempt to brute force it into place regardless of OS/distro and ignore errors doing so.